fuse/host_cgo.go

```diff
@@ -168,7 +168,9 @@ static void *cgofuse_init_fuse(void)
 
 	void *h;
 #if defined(__APPLE__)
-	h = dlopen("/usr/local/lib/libosxfuse.2.dylib", RTLD_NOW);
+	h = dlopen("/usr/local/lib/libfuse.2.dylib", RTLD_NOW); /* MacFUSE/OSXFuse >= v4 */
```

Comment:
Please change the comment to a single-line comment (i.e. from `/* MacFUSE/OSXFuse >= v4 */` to `// MacFUSE/OSXFuse >= v4`). Otherwise Go is confused from the nested `/* */` comments.